src/js/view/controlbar.js

```diff
@@ -307,6 +312,12 @@ define([
         onCastActive : function(model, val) {
             utils.toggleClass(this.elements.cast.element(), 'jw-off', !val);
         },
+        onAirplayAvailable : function(model, val) {
+            this.elements.airplay.toggle(val);
+        },
+        onAirplayActive : function(model, val) {
+            utils.toggleClass(this.elements.airplay.element(), 'jw-off', !val);
+        },
```

Comment:
OK. We'll leave it as existing tech-debt since we do the same with the cast buttons.